### PR TITLE
Remove unused argument metadata from es_read and make clearly private

### PR DESCRIPTION
**Pull request by dstandish**
Not used.

### CODE REVIEW

**Comment by jedcunningham** [comment]
We should consider `es_read` as private, yeah?

**Comment by dstandish** [comment]
> We should consider `es_read` as private, yeah?

Oh boy...
well... maybe technically we can't but 🤷

**Comment by dstandish** [comment]
airflow should make things all private unless a `@public` decorator is added.... default private everywhere

**Review by jedcunningham — APPROVED** [review]
My take is it's internal to the ES log handler, shouldn't be a problem to remove it.

**Comment by dstandish** [comment]
> My take is it's internal to the ES log handler, shouldn't be a problem to remove it.

well, if we're gonna do that, should we go all out and make it `_es_read` / `:meta private:`?

**Comment by eladkal** [comment]
> My take is it's internal to the ES log handler, shouldn't be a problem to remove it.
I'm fine with it but lets at least add entry about this in the change log that explains this decision. Simply log it at the top of the provider change log I will handle the style and place it in the correct place during release.

**Comment by jedcunningham** [comment]
> well, if we're gonna do that, should we go all out and make it `_es_read` / `:meta private:`?

Works for me.

**Comment by dstandish** [comment]
> I'm fine with it but lets at least add entry about this in the change log that explains this decision. Simply log it at the top of the provider change log I will handle the style and place it in the correct place during release.

OK thanks Elad

**Comment by dstandish** [comment]
ok @eladkal i added the docs note.  i also renamed it `_es_read`.  if you think the method rename goes too far LMK

File: airflow/providers/elasticsearch/CHANGELOG.rst [hunk 1]
```diff
@@ -27,6 +27,17 @@
 Changelog
 ---------
 
+5.1.0
+.....
+
+Misc
+~~~~
```
Comment (eladkal) [inline airflow/providers/elasticsearch/CHANGELOG.rst]: please remove. we don't know yet if we release 5.1 or 6 :)
the note itself is enough :)
```suggestion
```

**Review by eladkal — APPROVED** [review]

**Pull request merged**